tests/src/test/java/alluxio/client/cli/fsadmin/command/ReportCommandIntegrationTest.java

```diff
@@ -26,14 +26,6 @@
  * Tests for report command.
  */
 public final class ReportCommandIntegrationTest extends AbstractFsAdminShellTest {
-  @Test
-  public void masterNotRunning() throws Exception {
```

Comment:
this test is still applicable, we just have to use `assertThat(output, containsString(expected))` now